src/core_modules/capture-core/components/CardList/CardListItem.component.js

```diff
@@ -10,14 +10,14 @@ import type { CardDataElementsInformation, SearchResultItem } from '../Pages/Sea
 import type { DataElement } from '../../metaData';
 import { availableCardListButtonState, enrollmentTypes } from './CardList.constants';
 import { ListEntry } from './ListEntry.component';
+import { dataElementTypes } from '../../metaData';
 
 type OwnProps = $ReadOnly<{|
     item: SearchResultItem,
     currentSearchScopeName?: string,
-    currentProgramId?: string,
-    getCustomTopElements?: ?(props: Object) => Element<any>,
-    getCustomBottomElements?: ?(props: Object) => Element<any>,
-    imageDataElement: DataElement,
+    currentProgramId: ?string,
+    getCustomBottomElements?: (props: Object) => Element<any>,
+    profileImageDataElement: DataElement,
```

Comment:
while we're at it, maybe this can be of the same type as dataElements below? (singular). Also, this won't always have a value.

Reply:
You mean to define an explicit type for it that will describe exactly what is in the `profileImageDataElement`?

Reply:
yes 👍